bokehjs/src/coffee/models/widgets/multiselect.coffee

```diff
@@ -38,14 +39,25 @@ export class MultiSelectView extends InputWidgetView
       if values[el.attr('value')]
         el.attr('selected', 'selected')
     )
+    # Note that some browser implementations might not reduce
+    # the number of visible options for size <= 3.
+    @$el.find('select').attr('size', this.model.size)
 
   change_input: () ->
+    # Haven't checked if :focus selector works for IE <= 7
+    is_focused = @$el.find('select:focus').size()
```

Comment:
More idiomatic approach would be `is_focused = @$el.find('select').is(':focus')`.